components/sp-seamless-provision/src/main/java/com/wso2telco/carbon/identity/oauth2/token/handlers/clientauth/SeamlessProvisionClientAuthHandler.java

```diff
@@ -206,5 +218,23 @@ private MobileConnectConfig getMobileConnectConfig() {
         mobileConnectConfigs = configurationService.getDataHolder().getMobileConnectConfig();
         return mobileConnectConfigs;
     }
-
+   
+	public void setPublishingDataForError(OAuthTokenReqMessageContext tokReqMsgCtx)
+					throws IdentityOAuth2Exception {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.wso2.carbon.identity.oauth2.IdentityOAuth2Exception', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)